.nvmrc

```diff
@@ -1 +1 @@
-14
+16
```

Comment:
This PR should probably be rebased on the current develop as it's out of date with the current version of Node, deps, etc.